debian/setup.sh

```diff
@@ -0,0 +1,35 @@
+# !/bin/bash
+set -eu
+
+# Function to get the current Ubuntu version
+get_ubuntu_version() {
+    lsb_release -i -s 2>/dev/null
+}
+
+if [[ $(get_ubuntu_version) != "Ubuntu" ]]; then
```

Comment:
Is this for Ubuntu only? Or also Debian?

Reply:
technically should work for both, just copied this from Panda team though